uxarray/grid/grid.py

```diff
@@ -409,10 +427,46 @@ def Mesh2_face_x(self) -> xr.DataArray:
             _populate_centroid_coord(self)
         return self._ds['Mesh2_face_x']
 
+    @property
+    def Mesh2_face_cart_x(self) -> xr.DataArray:
+        """Coordinate ``Mesh2_face_cart_x``, which contains the Cartesian y of
+        each face center.
+
+        Dimensions (``nMesh2_face``)
+        """
+        if "Mesh2_face_cart_x" in self._ds:
+            return self._ds["Mesh2_face_cart_x"]
+        else:
+            return None
+
+    @property
+    def Mesh2_edge_x(self) -> xr.DataArray:
+        """Coordinate ``Mesh2_edge_x``, which contains the longitude of the
+        center of each edge.
+
+        Dimensions (``nMesh2_edge``)
+        """
+        if "Mesh2_edge_x" in self._ds:
+            return self._ds["Mesh2_edge_x"]
+        else:
+            return None
+
+    @property
+    def Mesh2_edge_cart_x(self) -> xr.DataArray:
+        """Coordinate ``Mesh2_edge_cart_x``, which contains the Cartesian x
+        location of each edge center.
```

Comment:
```suggestion
        """Coordinate ``Mesh2_edge_cart_x``, which contains the Cartesian x of
        each edge center in meters.
```

Reply:
For remaining properties, please make the docstrings consistent.  Either include or exclude "location" and always include the units.  There are some small inconsistencies in what follows.

Reply:
Good catch!